gnomad_constraint/plots/generate_constraint_plots.R

```diff
@@ -243,3 +309,226 @@ for (version in names(datasets)) {
     ggsave(p, filename = plot_path, dpi = 300, width = 8, height = 6, units = "in")
   }
 }
+
+
+####################################################################
+####################################################################
+# Plot LOEUF decile change from v2 to v4
+####################################################################
+####################################################################
+# Filter to MANE Select Ensembl transcripts that are present in both v2 and v4
+# and that have no constriant flags
+decile_data <- filter(
+  constraint_data,
+  (.data$v2 == TRUE &
+    .data$v4 == TRUE &
+    .data$constraint_flags == "[]" &
```

Comment:
If using other suggestion
```suggestion
    .data$constraint_flags.v4 == "[]" &
```